css/home.css

```diff
@@ -0,0 +1,774 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  overflow: hidden;
```

Comment:
- [ ] Please know that because of this line, the website disabled all scrolling and I couldn't scroll to check the content, kindly remove this, I know you used this because you had a horizontal scroll bar, but you should fix the issue that caused the horizontal scroll bar to appear rather than hiding it, I will try to pinpoint the issue in other comments. 😉